pypesto/store/hdf5.py

```diff
@@ -96,3 +98,35 @@ def write_int_array(f: h5py.Group,
     """
     dset = f.create_dataset(path, (len(values),), dtype=dtype)
     dset[:] = values
+
+
+def write_dict(
```

Comment:
I actually think it would be much easier to have that config just be a str, then one has absolutely no trouble of storing it ... what do you think?

Reply:
you mean instead of 
```
dict = {'key1': value1, 'key2' : value2}
```
rather 
```
dict = "key1: value1\nkey2: value2"
```
?

Reply:
could perhaps be a bit "problematic" with the aggregated objective but yes in principle that should work.

Reply:
you could also have them here always return dicts (either to append to), and just convert to string before saving.